server/athenian/api/__init__.py

```diff
@@ -95,7 +96,18 @@ def __init__(self, mdb_conn: str, sdb_conn: str, ui: bool,
 
     def create_app(self) -> aiohttp.web.Application:
         """Override the base class method to inject middleware."""
-        return aiohttp.web.Application(middlewares=[self.with_db])
+        middlewares=[self.with_db]
+        if not AuthConfigured():
+            # TODO: always require auth, or add a flag to assert that it's enabled
+            logging.getLogger(__package__).warning("API authentication is disabled; set AUTH0_DOMAIN and AUTH0_AUDIENCE")
+        else:
+            auth0 = Auth0(whitelist=[
+                r'/v1/openapi.json',
+                r'/v1/ui*',
+                r'/status*',
```

Comment:
`/status` is polled by the Google Cloud health checker, we cannot wall it...

Reply:
This is a whitelist (stuff not walled by auth).

Reply:
Oops sorry, I confused the lists! Then it should be fine as-as.